app/src/main/java/com/pavesid/androidacademy/di/AppModule.kt

```diff
@@ -0,0 +1,19 @@
+package com.pavesid.androidacademy.di
+
+import com.pavesid.androidacademy.repositories.MoviesRepository
+import com.pavesid.androidacademy.repositories.remote.MoviesRemoteRepository
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
```

Comment:
better to application component

Reply:
![image](https://user-images.githubusercontent.com/10943612/101901633-650d5980-3bc2-11eb-9111-b3d75a0afff2.png)

I'm not sure